openedx/features/learner_profile/__init__.py

```diff
@@ -11,3 +11,7 @@
 # Waffle flag to show achievements on the learner profile.
 # TODO: LEARNER-2443: 08/2017: Remove flag after rollout.
 SHOW_ACHIEVEMENTS_FLAG = WaffleFlag(WAFFLE_FLAG_NAMESPACE, 'show_achievements')
+
+# Waffle flag for increasing the visibility of the profile page, including all messaging necessary to show it.
+# TODO: LEARNER-2554: 09/2017: Remove flag once message is no longer needed.
+BOOST_PROFILE_VISIBILITY = WaffleFlag(WAFFLE_FLAG_NAMESPACE, 'boost_visibility')
```

Comment:
I don't see the change to make the header obey this Waffle flag. Are you planning to add that to the other PR once this merges?

Reply:
We discussed this during standup. It's a pain to try to wire up the header changes to a waffle flag since it touches so many files. We're going to merge this in first and then merge the header changes when they're ready.

Reply:
In that case I'd rename the flag to something like SHOW_NEW_PROFILE_MESSAGE. The comment and name as written don't fit with how it is going to be used.